libs/langchain/langchain/callbacks/openai_info.py

```diff
@@ -1,8 +1,14 @@
-"""Callback Handler that prints to std out."""
-from typing import Any, Dict, List
+"""Callback Handler that collects token usage from OpenAI models."""
+import warnings
+from typing import TYPE_CHECKING, Any, Dict, List, Tuple, Union
+from uuid import UUID
 
+from langchain.adapters.openai import convert_message_to_dict
```

Comment:
let's move this method more general if we want to use it here. something like `langchain/utils/openai.py`. and then we don't have to update check_imports

Reply:
I am not sure it is possible to do that without changing the check_imports. The function requires models from schema module but it is not allowed to be imported from utils. I made a commit (which fails on linting) and it touches a handful of files (and I am sure these changes will break some users' code).
What do you think is better:
1) Change the check_imports to allow utils import schema and move the function
2) Revert it to my original version and may be change in a separate PR later if it will cause troubles
?